creator-node/default-config.json

```diff
@@ -33,8 +33,8 @@
   "spOwnerWalletIndex": 1,
   "spOwnerWallet": "",
   "debounceTime": 30000,
-  "discoveryProviderWhitelist": "http://docker.for.mac.localhost:5000",
-  "identityService": "http://docker.for.mac.localhost:7000",
+  "discoveryProviderWhitelist": "http://audius-disc-prov_web-server_1:5000",
+  "identityService": "http://audius-identity-service_identity-service_1:7000",
   "userBlacklist": "",
   "trackBlacklist": "",
   "dataRegistryAddress": "",
```

Comment:
@hareeshnagaraj not letting me comment on lines 41 and 43, but those use docker.for.mac.localhost